portfolio/src/main/webapp/script.js

```diff
@@ -136,15 +136,15 @@ function loadComments(path) {
  * @param {string} url The url of an image to add to the comment, this may be
  *     undefined
  */
-function addComment(name, content, url) {
+function addComment(name, content, blobKey) {
```

Comment:
There are some other places in this file where you still refer to the `blobKey` parameter as a URL.